docs/conf_zh_CN.py

```diff
@@ -0,0 +1,185 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import json
+import shutil
+from sphinx.util import logging
+from json import JSONDecodeError
+from sphinx.ext.autodoc import ClassDocumenter
+
+
+sys.path.insert(0, os.path.abspath('doxyrest-sphinx'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'OpenVINO™'
+copyright = '2022, Intel®'
+author = 'Intel®'
+
+language = 'en'
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx_inline_tabs',
+    'sphinx_copybutton',
+    'sphinx_panels',
+    'doxyrest',
+    'cpplexer',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx_sitemap'
+]
+
+html_baseurl = 'https://docs.openvino.ai/latest/'
+
+sitemap_url_scheme = "{link}"
+
+html_favicon = '_static/favicon.ico'
+autodoc_default_flags = ['members']
+autosummary_generate = True
+autosummary_imported_members = True
+
+html_logo = '_static/logo.svg'
+html_copy_source = False
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+panels_add_bootstrap_css = False
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "openvino_sphinx_theme"
+
+html_theme_path = ['_themes']
+
+html_theme_options = {
+    "navigation_depth": 6,
+    "show_nav_level": 2,
+    "use_edit_page_button": True,
+    "github_url": "https://github.com/openvinotoolkit/openvino",
+    "footer_items": ["footer_info"],
+    "show_prev_next": False,
+}
+
+html_context = {
+    'current_language': '中文',
+    'languages': (('English', '/latest'), ('中文', '/cn/latest')),
```

Comment:
```suggestion
    'languages': (('English', '/latest'), ('简体中文', '/zh_CN/latest')),
```

Reply:
```suggestion
    'languages': (('English', '/latest'), ('简体中文', '/cn/latest')),
```
Same as before... but we should keep the language name, I assume?